features/step_definitions/login_and_registration_steps.rb

```diff
@@ -0,0 +1,58 @@
+Given("I am not signed in") do
```

Comment:
Not blocking - Can we name this file `sign_up_and_sign_in_steps` to match the other bits of the codebase?